modin/pandas/test/test_io.py

```diff
@@ -1448,18 +1442,36 @@ def test_read_parquet_pandas_index(self):
         pandas_df = pandas.DataFrame(
             {
                 "idx": np.random.randint(0, 100_000, size=2000),
+                "idx_categorical": pandas.Categorical(["y", "z"] * 1000),
+                # Can't do interval index right now because of this bug fix that is planned
+                # to be apart of the pandas 1.5.0 release: https://github.com/pandas-dev/pandas/pull/46034
+                # "idx_interval": pandas.interval_range(start=0, end=2000),
+                "idx_datetime": pandas.date_range(start="1/1/2018", periods=2000),
+                "idx_periodrange": pandas.period_range(
+                    start="2017-01-01", periods=2000
+                ),
                 "A": np.random.randint(0, 100_000, size=2000),
                 "B": ["a", "b"] * 1000,
                 "C": ["c"] * 2000,
             }
         )
-        try:
-            pandas_df.set_index("idx").to_parquet(unique_filename)
-            # read the same parquet using modin.pandas
-            df_equals(
-                pd.read_parquet(unique_filename), pandas.read_parquet(unique_filename)
+        # Older version of pyarrow do not support Arrow to Parquet
+        # schema conversion for duration[ns]
+        if sys.version_info >= (3, 7):
```

Comment:
The comment talks about pyarrow but you check Python version... feels weird. Can we explicitly check something like `version.parse(pyarrow.__version__) >= version.parse("some.known.version")` instead?

Reply:
Yes, that's fair. I did that for `fsspec` just now so I'll do the same for `pyarrow`. Thanks for pointing it out.